README.md

```diff
@@ -197,6 +197,8 @@ The following environment variables are supported:
   wallet. Useful for scripting or other non-interactive commands, but
   use with care.
 
+* `HELIUM_WALLET_SEED_WORDS` - Seed words to use when restoring an
+  existing `mobile` or `bip39` wallet from a mnemonic word list.
```

Comment:
After #250 lands, you should remove the part about `mobile` and `bip39`. It's just going to be a mnemonic word list.

Reply:
@ke6jjj See my comment above?

Reply:
yes! I am so behind on my PR tending. Apologies, @kellybyrd! let's do it.